### PR TITLE
Adding redis helper for redis resiliency workflow

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

 - Adding redis helper for redis resiliency workflow
 - Redis Dashboard
 - Cmd probe checker
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by oumkale** [comment]
Screenshot from 2022-01-03 14-57-25
Screenshot from 2022-01-03 14-57-29

**Comment by oumkale** [comment]
Screenshot from 2022-01-03 14-57-29
Screenshot from 2022-01-03 17-25-33

File: custom/workflow-helper/redis-helper/load-gen/redisLoad.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from pyredis import Client
+import time, string, random, os
+
+host = os.getenv("REDIS_HOST", "redis.redis.svc")
+
+# DBDetails is performing io operations on redis db
+class DBDetails(object):
+    def __init__(self):
+        self.client       = Client(host=host)
```
Comment (avaakash) [inline custom/workflow-helper/redis-helper/load-gen/redisLoad.py:9]:
```suggestion
        self.client = Client(host=host)
```

File: custom/workflow-helper/redis-helper/load-gen/redis-cmd.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from pyredis import Client
+import time, string, random, os
+
+# DBDetails is performing io operations on redis db
+class DBDetails(object):
+    def __init__(self):
+        self.client       = Client(host=os.getenv("REDIS_HOST", "redis.redis.svc"))
```
Comment (avaakash) [inline custom/workflow-helper/redis-helper/load-gen/redis-cmd.py:7]:
```suggestion
        self.client = Client(host=os.getenv("REDIS_HOST", "redis.redis.svc"))
```

**Review by avaakash — CHANGES_REQUESTED** [review]

**Review by avaakash — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**